src/views/ScenarioManager/labels.js

```diff
@@ -10,9 +10,9 @@ export const getScenarioManagerLabels = (t) => ({
   failed: t('commoncomponents.scenariomanager.treelist.node.status.failed'),
   created: t('commoncomponents.scenariomanager.treelist.node.status.created'),
   delete: t('commoncomponents.scenariomanager.treelist.node.action.delete'),
-  edit: t('commoncomponents.scenariomanager.treelist.node.action.edit'),
   scenarioRename: {
     title: t('commoncomponents.dialog.create.scenario.input.scenarioname.label'),
+    edit: t('commoncomponents.scenariomanager.treelist.node.action.edit'),
```

Comment:
This change might be considered as a breaking change, but I'm not sure if we can (and if we want to) keep supporting both possible versions. Maybe we'll just release a new major version and keep the code as simple as possible

Reply:
As we tried to avoid a new major version with `canUpdate/canRename` props, maybe, it's better to duplicate this label as well ? it's not very clean but at least, we don't release a new major version because of one label :thinking:

Reply:
Ok, let's do this :slightly_smiling_face: